src/Themes.scss

```diff
@@ -0,0 +1,1392 @@
+                    fill: #67717E;
+                }
+                path.triangle {
+                    fill: #30444E;
+                }
+            }
+        }
+    }
+
+    button.btn.PlayButton--disabled, button.btn.StopButton--disabled {
+        background-color: #D5D8DD;
+
+        svg {
+            path.inner-circle {
+                fill: #D5D8DD;
+            }
+            circle, path.triangle {
+                fill: #9DA4AF;
+            }
+        }
+    }
+
+    .ProgramSpeedController__container svg path {
+        fill: #67717E;    
+    }
+
+    .ProgramSpeedController__slider {
+        &::-webkit-slider-runnable-track {
+            background-color: #67717E;
+            border: 2px solid #67717E;
+        }
+
+        &::-webkit-slider-thumb {
+            border: 2px solid #67717E;
+            background-color: white;
+            box-shadow: -8rem 0 0 8rem white;
+        }
+
+        &::-moz-range-thumb {
+            border-radius: 0;
+            border: 2px solid black;
+            background-color: white;
+            box-shadow: -8rem 0 0 8rem white;
+        }
+
+        &::-moz-range-progress {
+            background-color: white; 
+        }
+        &::-moz-range-track {
+            background-color: #67717E;
+        }
+
+        &::-ms-fill-lower {
+            background-color: white; 
+        }
+
+        &::-ms-fill-upper {  
+            background-color: #67717E;
+        }
+
+        &:hover {
+            &::-webkit-slider-thumb {
+                background-color: #9DA4AF;
+                box-shadow: -8rem 0 0 8rem #9DA4AF;
+            }
+            &::-moz-range-thumb {
+                background-color: #9DA4AF;
+                box-shadow: -8rem 0 0 8rem #9DA4AF;
+            }
+    
+            &::-moz-range-progress {
+                background-color: #9DA4AF; 
+            }
+    
+            &::-ms-fill-lower {
+                background-color: #9DA4AF; 
+            }
+        }
+
+        &:focus {
+            outline-color: #505862;
+        }
+    }
+
+
+    button.btn.PlayButton--play.PlayButton--disabled {
+        background-color: inherit;
+        cursor: default;
+    
+        svg {
+            path.triangle {
+                fill: #30444E;
+            }
+        }
+    }
+
+    button.btn.ShareButton {
+        background-color: #F1F2F4;
+        color: black;
+
+        &:hover {
+            background-color:#D5D8DD;
+        }
+
+        &:focus {
+            box-shadow:
+                0 0 0 0.15rem #d5d8dd,
+                0 0 0 0.25rem #505862;
+        }        
+    }
+}
+
+/* High Contrast "Mono" Theme */
+body.contrast-theme {
+    background-color: black;
+
+    .App__container {
+        background-color: black;
+    }
+    
+    .App__header {
+        background-color: black;
+        color: white;
+    }
+
+    .ThemeSelector {
+        button.btn {
+            background-color: white;
+            color: black;
+    
```

Comment:
ThemeSelector for the contrast theme changes its size on hover, because border is applied on hover. We can prevent this by applying border to its base style with same background-color.

```
button.btn {
            background-color: white;
            color: black;
            border: 2px solid white;
```

Reply:
I followed a similar strategy and removed all the padding hints as well.  The size remains stable in all themes now.